hivemind/modules/api/models/profile.py

```diff
@@ -46,11 +64,35 @@ class Meta:
 
         ordering = ['id', 'school_id', 'last_name']
 
+    def get_full_name(self):
+        """
+        Returns the full name of the person
+        """
+        if self.preposition:
+            return f"{self.first_name} {self.preposition} {self.last_name}"
+
+        return f"{self.first_name} {self.last_name}"
+
+    def get_last_name(self):
+        """
+        Returns the last name of the person.
+        """
+        if self.preposition:
+            return f"{self.last_name}, {self.preposition}"
+
+        return self.last_name
+
+    def get_first_name(self):
```

Comment:
I don't really see the need for this method. As it has no check nor modification to what `self.first_name` would return.